docs/events.md

```diff
@@ -575,3 +575,17 @@ A `batch end <type>` event is also triggered.
     commands: []
 }
 ~~~
+
+
+**cap ls**, **cap ack**, **cap nak**, **cap list**, **cap new**, **cap del**
+
+Triggered for each `CAP` command, lists the sent capabilities list.
+
+~~~javascript
+{
+    command: 'LS',
```

Comment:
It looks like this `ls` should actually be lowercase

Reply:
It's `command.params[1]`, won't be lowercase here.